acceleration/Dockerfile.spiceai

```diff
@@ -0,0 +1,17 @@
+FROM rust as build
+
+RUN apt-get update && apt-get install -yqq git
+
+RUN git clone https://github.com/spiceai/spiceai.git
+
+RUN cd spiceai && make -C bin/spiced
+
+FROM debian
+
+RUN apt-get update && apt-get install -yqq libssl-dev sqlite3
+
+COPY --from=build /spiceai/target/release/spiced /usr/local/bin
```

Comment:
Could we have this pull the latest release from GitHub instead of compiling? That would cut down on the time needed to build this. (We could also update in the next release to pull from the official docker image)

Reply:
Actually for this to work properly on M1 macs, we need to build a Linux aarch64 version of spiced

Reply:
Please resolve / address in next iteration.  Closing comment to merge.